webapp/webpack.common.js

```diff
@@ -66,6 +66,22 @@ function makeCommonConfig() {
                     ],
                     exclude: [/node_modules/],
                 },
+                {
+                    test: /\.(png|eot|tiff|svg|woff2|woff|ttf|gif|mp3|jpg)$/,
```

Comment:
Why are we exposing here more than we need? the png, tiff, gif, mp3 and jpg files are not needed yet, why we add them upfront if it is not explicitly needed?

Reply:
Done.